cmd/broker/provisioning_test.go

```diff
@@ -223,7 +223,6 @@ func TestProvisioning_HappyPathSapConvergedCloud(t *testing.T) {
 	suite.AssertKymaResourceExists(opID)
 	suite.AssertKymaAnnotationExists(opID, "compass-runtime-id-for-migration")
 	suite.AssertKymaLabelsExist(opID, map[string]string{"kyma-project.io/region": "eu-de-1"})
-	suite.AssertKymaLabelNotExists(opID, "kyma-project.io/platform-region")
```

Comment:
Why did you removed this assertion?
I assume that it was failing. So why it was failing?
Did we changed the behavior recently?

Reply:
Region mappings for sap-converged-cloud enforce passing region in the path (see above). This in turn adds label `kyma-project.io/platform-region`.

Reply:
Got it.